internal/parsers/markdown.go

```diff
@@ -73,7 +74,7 @@ func ExtractScenarioTitleFromAst(node ast.Node, source []byte) (string, error) {
 	return header, nil
 }
 
-var expectedSimilarityRegex = regexp.MustCompile(`<!--\s*expected_similarity=\s*(\d+\.?\d*)\s*-->`)
+var expectedSimilarityRegex = regexp.MustCompile(`<!--\s*expected_similarity=\s*(\d+\.?\d*)|("\.")\s*-->`)
```

Comment:
Could we implement at least two tests for `ExtractCodeBlocksFromAst` that ensures that:

1. The expected similarity for similarity thresholds are parsed correctly
2. The expected similarity for regex strings are parsed correctly

If you don't have the bandwidth to do so, I could contribute these tests directly to your PR.